src/applications/personalization/dashboard/actions/prescriptions.js

```diff
@@ -31,6 +32,16 @@ export function loadPrescriptions(options) {
       type: options.active ? 'LOADING_ACTIVE' : 'LOADING_HISTORY',
     });
 
+    if (environment.isLocalhost() || window.Cypress) {
```

Comment:
We probably don't want `window.Cypress` here or we won't be able to mock the API response in Cypress tests.